app/views/users/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<% content_for :title, "#{@user.name}' Blog" %>
+<section class="userDetailsSection">
+  <%= render "user", user: @user %>
+  <div class="bio">
+    <h3>Bio</h3>
+      <p class='bioDesc'><%= @user.bio %></p>
+  </div>
+  <br>
+  <div class='recentPosts'>
+    <% @user.recent_posts.each do |post| %>
+      <%= link_to user_post_path(@user, post) do %>
+        <%= render "posts/post", post: post %>
+      <% end %>
+    <% end %>
+  </div>
+  <%= link_to user_posts_path(@user) do %>
+    <button type="button" class="seeAllPosts">See all posts</button>
+  <% end %>
```

Comment:
- [ ] See all posts is redirecting to  post#index but list of posts all not show/ listed as it supposed to work.  fix it and make sure that you following the provided wireframe to fix the issues

Reply:
Thank you, bro, you can see the above and these highlighted functionalities in my local environment:


https://github.com/Fahim-Khan-P/blog/assets/74261465/76c791a2-094c-403a-8af5-58e8466ced31